packages/squarev2-integration/src/SquareV2PaymentMethod.tsx

```diff
@@ -0,0 +1,200 @@
+import { difference } from 'lodash';
```

Comment:
Do we need to worry about translations over here?

Reply:
Discussed offline and square's SDK will take care of that.

Reply:
AFAIK, Square's widget takes care of that. https://developer.squareup.com/reference/sdks/web/payments/objects/Payments#Payments.setLocale